library/std/src/sys/personality/gcc.rs

```diff
@@ -59,6 +59,9 @@ const UNWIND_DATA_REG: (i32, i32) = (0, 1); // R0, R1 / X0, X1
 #[cfg(target_arch = "m68k")]
 const UNWIND_DATA_REG: (i32, i32) = (0, 1); // D0, D1
 
+#[cfg(target_arch = "csky")]
+const UNWIND_DATA_REG: (i32, i32) = (0, 1); // R0, R1
```

Comment:
Where can I cross-reference the value for this and `unwinder_private_data_size` against the official compiler?

Reply:
Also given that libstd isn't supported yet, maybe omit this for now?

Reply:
https://github.com/llvm/llvm-project/blob/28555793b1e58ee2e147ea62de42a9583fadbc5a/llvm/lib/Target/CSKY/CSKYISelLowering.cpp#L1257-L1265